module/documents/actor.mjs

```diff
@@ -1375,9 +1376,19 @@ export default class CrucibleActor extends Actor {
 
     // Confirmation dialog
     if ( dialog ) {
+      let content = `<p>Spend 1 Talent Point to purchase <strong>${talent.name}</strong>?</p>`;
+
+      if ( warnUnusable ) {
+        // Add warning to content if acquiring Gesture or Inflection without Rune
+        if ( (talent.system.gesture || talent.system.inflection) && !this.items.find(i => (i.type === "talent" && i.system.rune)) ) {
+          content += `<div class="notification warning">
+            ${game.i18n.localize(`TALENT.WARNINGS.RequiresRune${talent.system.inflection ? "Inflection" : "Gesture"}`)}
+          </div>`;
+        }
+      }
```

Comment:
I think rather than encoding special logic for testing this in this particular `addTalent` method, we should delegate this out as something like `CrucibleActor#canUtilizeTalent(talent)` which could return `true` or throw an `Error` with a message to display as the reason that purchasing is not possible. 

So then this would look like:

```js
// Maybe warn if the talent is not able to be purchased
try {
  const canUse = this.canUtilizeTalent(talent);
  if ( (canUse === false) && warnUnusable ) {
    content += `<div class="notification warning">You cannot use this talent.</div>`;
  }
} catch(err) {
  if ( warnUnusable ) {
    content += `<div class="notification warning">${err.message}</div>`;
  }
}
```